entity-framework/core/modeling/relationships.md

```diff
@@ -135,6 +135,12 @@ You can use the string overload of `HasForeignKey(...)` to configure a shadow pr
 
 [!code-csharp[Main](../../../samples/core/Modeling/FluentAPI/Samples/Relationships/ShadowForeignKey.cs#Sample)]
 
+### Without Navigation Property
```

Comment:
@AndriySvyryd @ajcvickers can you think of any pitfalls worth mentioning here?

Reply:
Looks good. The only pitfall I can think of is the obvious one--there's no navigation properties to use!